src/main/java/org/opensearch/remoteprocessor/processors/RemoteSearchResponseProcessor.java

```diff
@@ -49,15 +51,27 @@ public class RemoteSearchResponseProcessor extends AbstractProcessor implements
     private final String processorName;
     private final RemoteProcessorServiceGrpc.RemoteProcessorServiceStub rpsRpcClient;
 
-    protected RemoteSearchResponseProcessor(String tag, String description, boolean ignoreFailure, String endpoint, String processorName) {
+    protected RemoteSearchResponseProcessor(String tag, String description, boolean ignoreFailure, String endpoint, String processorName)
```

Comment:
can we switch to insecure if `endpoint` starts with 'http'?

Reply:
sure!